dali/kernels/normalize/normalize_cpu.h

```diff
@@ -139,13 +154,13 @@ struct NormalizeCPU {
         param_strides[i] = 0;  // reduced dim - use the same parameter slice for all data slices
     }
     NormalizeAxis(0, output_, input_, data_strides.data(),
-                  mean_, inv_stddev_, param_strides.data());
+                  mean_, scale_, param_strides.data());
   }
 
   void NormalizeAxis(int axis,
     Out *out, const In *in,
     const int64_t *data_strides,
-    const MeanStdDev *mean, const MeanStdDev *stddev,
+    const Param *mean, const Param *scale,
```

Comment:
Why `mean` and `scale` are arguments and not obtained silently from object like `shift_`

Reply:
Because they are nontrivially advanced using param_strides - not just forwarded like shift would be.